Java/E-montazysta/src/main/java/com/emontazysta/model/dto/ElementDto.java

```diff
@@ -15,9 +18,13 @@
 public class ElementDto {
 
     private Long id;
+    @NotBlank(message = "Name cannot be empty")
     private String name;
+    @NotBlank(message = "Code cannot be empty")
```

Comment:
```suggestion
```
Code is created on backend while creating new element.
https://github.com/s20146/PRO/blob/23a8b9a0ff4f1cd696ca79848103ed664bd4da22/Java/E-montazysta/src/main/java/com/emontazysta/service/impl/ElementServiceImpl.java#L52